src/neo4j_graphrag/llm/openai_llm.py

```diff
@@ -120,28 +127,35 @@ async def ainvoke(
         message_history: Optional[list[LLMMessage]] = None,
         system_instruction: Optional[str] = None,
     ) -> LLMResponse:
-        """Asynchronously sends a text input to the OpenAI chat
-        completion model and returns the response's content.
-
-        Args:
-            input (str): Text sent to the LLM.
-            message_history (Optional[list]): A collection previous messages, with each message having a specific role assigned.
-            system_instruction (Optional[str]): An option to override the llm system message for this invokation.
-
-        Returns:
-            LLMResponse: The response from OpenAI.
-
-        Raises:
-            LLMGenerationError: If anything goes wrong.
+        """
```

Comment:
The previous docstring is still valid, but now we return content OR tool calls.

Reply:
https://github.com/neo4j/neo4j-graphrag-python/pull/268/commits/4801ed02b915fc16635b841cec03ee7d194c932f